wiki/mapping/README.md

```diff
@@ -223,7 +226,9 @@ See the [Playtesting](/mapping/#playtesting) section for instructions on testing
 
 #### How to Release a Map
 
-1. [Create an account](https://beatsaver.com/auth/register) on BeatSaver. If you are not receiving a verification email, drop into `#mapping-discussion` for assistance.
+1. [Create an account](https://beatsaver.com/auth/register) on BeatSaver.  
+    - If you did not receive a verification email, fill out this [contact form](https://beatsaver.com/contact).  
+    **Note** that a valid username *and* email used to sign up is needed to complete the verification!
```

Comment:
`**Note** that a valid username *and* email used to sign up is needed to complete the verification!`
The wording of this sentence is confusing.
I'd suggest
`**Note** when you sign up you must use a valid username *and* email. Otherwise your account cannot be verified!`

Reply:
Ah, its based off of the pinned message from the jellyfish for their contact form. since one person didn't give the correct email to the username. 
This note is for after they signed up but are not verified to upload

Reply:
`**Note** you must provide a valid username *and* the email used to sign up in the form to complete the verification!`

Maybe this?

Reply:
Oh, I misunderstood what that sentence was supposed to mean!
Yeah, that makes more sense. Looks good to me